src/parser/csql_grammar.y

```diff
@@ -28382,4 +28390,25 @@ pt_fill_conn_info_container(PARSER_CONTEXT *parser,  int buffer_pos, container_1
                 assert(0);
                 break;
     }
+}
+
+static bool pt_ct_check_select (char* p, char *perr_msg)
+{  
+   perr_msg[0] = 0x00;
+   while (*p == ' ' || *p == '(' || *p == '\t' || *p == '\r' || *p == '\n')
+     {
+        p++;
+     }
+
+   if(*p)
+   {
+        if( strncasecmp(p, "SELECT", 6) == 0 )
```

Comment:
This could be a problem with the sql below.
```
WITH table AS
(
  SELECT ...
)
SELECT * FROM table;
```
It is necessary to check the various sql usage.

Reply:
First of all, only the case of starting with SELECT is supported.
The WITH clause is not supported.